### PR TITLE
Add password reset option

### PR DESCRIPTION
**Pull request by kmets89**
Discussed in ticket 517551

### CODE REVIEW

File: src/software/js-samples/importUsers.html [hunk 1]
```diff
@@ -186,6 +188,17 @@ <h2>Help</h2>
...
+				// $(passwordResetCheckbox).change(function(){
+				// 	if ($(this).is(":checked")){
+				// 		document.getElementById("import-password").disabled = true;
+				// 		//$("#import-password").disabled = true;
+				// 	}
+				// 	else{
+				// 		//$("#import-password").disabled = false;
+				// 		document.getElementById("import-password").disabled = false;
+				// 	}
+				// });
```
Comment (beefo) [inline src/software/js-samples/importUsers.html]: Shouldn't leave commented code.

File: src/software/js-samples/importUsers.html [hunk 2]
```diff
@@ -293,11 +303,22 @@ <h2>Help</h2>
...
-									}
+										activeTo: "2050-01-01T00:00:00.000Z"
+									};
+							if (keys.length > 0){
+								newUser.driverGroups =  groups;
+								newUser.viewDriversOwnDataOnly =  false;
+								newUser.keys = keys;
+								newUser.isDriver = true;
+							}
+							if ($(passwordResetCheckbox).is(":checked"))
```
Comment (beefo) [inline src/software/js-samples/importUsers.html]: Should include open/close braces.

File: src/software/js-samples/importUsers.html [hunk 3]
```diff
@@ -293,11 +303,22 @@ <h2>Help</h2>
...
+								newUser.changePassword = true;
+							
+							console.log(newUser);
```
Comment (beefo) [inline src/software/js-samples/importUsers.html]: Need this console log?

**Comment by beefo** [comment]
Functionally tested and works well. Few comments for code cleanup.

**Comment by kmets89** [comment]
I made the fixes you mentioned.

**Pull request merged**